test/Header/SetCookieTest.php

```diff
@@ -343,6 +343,12 @@ public function testDoesNotAcceptCookieNameFromArbitraryLocationInHeaderValue()
         $this->assertNotEquals('leo_auth_token', $setCookieHeader->getName());
     }
 
+    public function testEmptyExpiresSection()
+    {
+        $cookie = 'Set-Cookie:path=/; Expires=; HttpOnly;';
+        $setCookieHeader = SetCookie::fromString($cookie);
```

Comment:
dont need to add try catch, you can use `@expectedException`, overall i will only update code with `assertIsNull`, your fix is not correct. Tests should not always have assertions, if test has passed without exceptions than it is ok, pass or fail